docs/DeveloperGuide.adoc

```diff
@@ -16,6 +16,8 @@ By: `Team T09-B3`      Since: `Feb 2018`      Licence: `MIT`
 
 == Setting up
 
+There are some requirements you will need to set up before getting started in contributing to ProgressChecker. Below lists the important key elements you will have to configure.
```

Comment:
set up requirement? will "There are some requirements you will need to follow to set up the app" be better?